src/GramatykiGrafoweAGH/visualization.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, List, Optional, Tuple
+
+import matplotlib.pyplot as plt
+import networkx as nx
+
+from GramatykiGrafoweAGH import Node, Graph
+
+node_colors = {
+    0: {
+        'E': '#fe0000',
+        'e': '#fe0000',
+    },
+    1: {
+        'E': '#4e81bd',
+        'I': '#c0504d',
+        'i': '#c0504d',
+    },
+    2: {
+        'E': '#93d051',
+        'I': '#f79645',
+        'i': '#f79645',
+    },
+    3: {
+        'E': '#feff00',
+        'I': '#d9d9d9',
+        'i': '#d9d9d9',
+    },
```

Comment:
What if we have more levels?

Reply:
For levels 4 and above, the colors for level 3 are used, see here: https://github.com/GramatykiGrafoweAGH/GramatykiGrafoweAGH/blob/task-1/src/GramatykiGrafoweAGH/viualization.py#L32